Mixpanel/MPABTestDesignerConnection.m

```diff
@@ -15,11 +15,13 @@
 #import "MPDesignerSessionCollection.h"
 #import "MPLogger.h"
 #import "MPSwizzler.h"
+#import "MPResources.h"
 
 NSString * const kSessionVariantKey = @"session_variant";
+static const int connectivityIndicatorSize = 50;
```

Comment:
sure, the reason is that I would rather have the flexibility for our devs to be able to change it as they see fit in the future. Furthermore, I would rather instantiate the window first rather than the uiimageview in terms of code cleanness. Lastly, that variable is called not only in the instantiation so if for some reason things move around or it is used in different places, I would rather it will have a value straight away than rely on runtime execution